### PR TITLE
[ME-828] Always override client with new config

### PR DESCRIPTION
**Pull request by davidnguyen2302**
https://fieldwire.atlassian.net/browse/ME-828

Previously, we only initialized the Client object (used to upload files for Filestack) when it's `null` and reuse the same object if it's not (even though there might have been a different config param passed in, i.e. config with a different region). 

Additionally, Filestack is currently being set on a different process as compared to our application (separate `android:process` is set in the AndroidManifest, this was to avoid crashes from Filestack to impact our application since it's not being maintained anymore), thus even though the Filestack Activity might be dismissed, it still retains all the references to the previous usage.

TLDR; Filestack reuses the same storage config once it sets even though we create a new FilestackPicker with new config every time, so removing the `client == null` logic lets the `initializeClient()` call to set the client every time so we don't reuse the old config.

### CODE REVIEW

**Review by tjaved970 — APPROVED** [review]

**Review by firoze — APPROVED** [review]

**Comment by mbalajee** [comment]
@davidnguyen2302 to master?

**Comment by davidnguyen2302** [comment]
> @davidnguyen2302 to master?

@mbalajee I'm assuming that's the main branch for this repo, once it's merged I can get the new commit hash code to update our imported version in our FW app, should I be committing to a different branch?

**Comment by mbalajee** [comment]
> > @davidnguyen2302 to master?
> 
> @mbalajee I'm assuming that's the main branch for this repo, once it's merged I can get the new commit hash code to update our imported version in our FW app, should I be committing to a different branch?



> > @davidnguyen2302 to master?
> 
> @mbalajee I'm assuming that's the main branch for this repo, once it's merged I can get the new commit hash code to update our imported version in our FW app, should I be committing to a different branch?

Nvm, I din't look at the repo.  All good.

**Review by mbalajee — APPROVED** [review]
